packages/react-components/react-progress/src/stories/Progress/ProgressAppearance.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import { Progress } from '@fluentui/react-progress';
+
+const intervalDelay = 100;
+const intervalIncrement = 0.01;
+
+const useStyles = makeStyles({
+  container: {
+    ...shorthands.padding('20px'),
+  },
+});
+
+export const Appearance = () => {
+  const styles = useStyles();
+  const [percentComplete, setPercentComplete] = React.useState(0);
+
+  React.useEffect(() => {
+    const id = setInterval(() => {
+      setPercentComplete((intervalIncrement + percentComplete) % 1);
+    }, intervalDelay);
+    return () => {
+      clearInterval(id);
+    };
+  });
```

Comment:
We probably shouldn't "animate" the normal progress bar like this in an example, since it's not a feature of the progress control.